python/thunder/rdds/timeseries.py

```diff
@@ -298,6 +297,21 @@ def normalize(self, baseline='percentile', window=None, perc=20):
             basefunc = lambda x: asarray([percentile(x[max(ix-left, 0):min(ix+right+1, n)], perc)
                                           for ix in arange(0, n)])
 
+        if method == 'window-fast':
+            if window%2:
+                left, right = ((window-1)/2, (window-1)/2)
+            else:
+                left, right = (window/2 - 1, window/2)
+
+            n = len(self.index)
+            ind_perc = round((window-1)*(perc/100)) 
+            
+            def basefunc(x):
+                left_pad = mean(x[:right])
```

Comment:
Remove unscores by replacing with `leftPad` and `rightPad`. See the new style guide (to be added in 0.5.0): https://github.com/freeman-lab/thunder/blob/master/python/doc/style_guide.rst